### DISCUSSION TITLE
is it possible connect a different backend?

### DISCUSSION THREAD

**Discussion by Khyretos** [owncast/owncast#3936] [Q&A]
Hello there, i have a quick question. i found a backend called "Oven Media Engine" and i would like to know how difficult it would be to connect a stream that is already being processed and send it to owncast. for example i have stream that is being run on ws://xxx.xxx.xxx.xx:3333/app/stream and i want to connect it directly to owncast, is that possible? owncast would not need to do any processing or anything. just show the footage. i ask this because i really like the frontend of owncast but the delay is too much for me to use it personally. is this possible?

**Comment by Khyretos** [comment]
i guess i just want to use it as a webrtc streamer.

**Comment by gabek** [comment]
This is not possible.

**Reply by Khyretos** [reply]
that is unfortunate. thank you for the quick awnser
